UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>I'd recommend splitting these up into multiple cells.</p><p> </p><p>For the learning goals, can you add some more specificity? For example, for "Download data" I presume (after having read the intro) this will specifically be GALEX spectra and we'll be using astroquery to download it from the MAST archive? [Haha, nope! I was wrong. I came back to this comment after reading through the rest of the tutorial.] </p><p> </p><p>It seems the learning goals might be more like....</p><ul><li>Load a catalog of star coordinates from a text file</li><li>Load a catalog of star coordinates from Vizier</li><li>Search GUVcat and Simbad for photometry of target stars given coordinates</li><li>Create a color-color diagram of stellar populations and interpret the results</li></ul><p> </p><p>That might not be extensive, there is probably more here that I am missing because the tutorial is very comprehensive, but those are the ones that stood out to me upon first review. </p><p> </p><p>As the reader I was also hoping for a bit more at the end of the introduction. I'm left wondering, of all thse cool things we can do with GALEX(look for variable, detect binaries, detect chromospheres), are we actually going to do one in the tutorial? Which one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>